robottelo/cli/sm_report.py

```diff
@@ -0,0 +1,31 @@
+"""
+Usage:
+    satellite-maintain report [OPTIONS] SUBCOMMAND [ARG] ...
+
+Parameters:
+    SUBCOMMAND       subcommand
+    [ARG] ...        subcommand arguments
+
+Subcommands:
+    generate         Generates the usage reports
+
+Options:
+    --output FILE    Output the generate report into FILE
+    -h, --help       print help
+
+"""
+
+from robottelo.cli.base import Base
+
+
+class Report(Base):
```

Comment:
Won't people mistake this with report template report generation? Because I did when I saw `module_target_sat.cli.Report.generate({'output': filename})` :)